packages/astro/src/core/config.ts

```diff
@@ -507,6 +522,8 @@ async function tryLoadConfig(
 		// Fallback to use Vite DevServer
 		const viteServer = await vite.createServer({
 			server: { middlewareMode: true, hmr: false },
+			optimizeDeps: { entries: [] },
+			clearScreen: false,
```

Comment:
I noticed the screen refreshes with a warning statement on our Vite server fallback. `entries` prevents cache warnings, and `clearScreen: false` preserves output!